common/guestchain/src/manager.rs

```diff
@@ -494,4 +513,51 @@ fn test_generate() {
         false,
     )
     .unwrap();
+
+    let update_chain_config = UpdateConfig {
+        min_validators: Some(
+            NonZeroU16::new((mgr.validators().len() + 1) as u16).unwrap(),
+        ),
```

Comment:
```suggestion
        min_validators: NonZeroU16::new((mgr.validators().len() + 1) as u16),
```
is also fine. Up to you if you want to be more explicit or more concise.